.browserslistrc

```diff
@@ -5,8 +5,10 @@
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
-last 2 versions
+last 1 Chrome version
```

Comment:
Why is this changing to support the last 2 versions of everything except for Chrome which is our most popular browser?

Reply:
Just copying the default settings from the Angular CLI generated ones. My guess is that Firefox and Chrome auto-updates enough that the last 1 major version is enough